WorkflowFunctionAdminBundle/Resources/config/resolve_document.yml

```diff
@@ -0,0 +1,2 @@
+resolve_target_documents:
+        OpenOrchestra\WorkflowFunction\Model\ReferenceInterface: OpenOrchestra\ModelBundle\Document\ContentType
```

Comment:
Why this relation ? 

Isn't there a Reference class that implements the ReferenceInterface